modules/add.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-cycle */
+import Book from '../index.js';
+import Books from './books.js';
+import luxon from './luxon.js';
+
+const books = new Books();
+
+const awesomeBooks = document.createElement('div');
+awesomeBooks.innerHTML = '';
+
+document.getElementById('date').innerText = luxon.DateTime.now().setLocale('en-US').toLocaleString(luxon.DateTime.DATETIME_FULL_WITH_SECONDS);
+
+function addToList(bookObj, index) {
```

Comment:
It will be better to make this function an `es6 syntax` function. Remember we need to use `es6` for this milestone. 

**HINT**
```
const timeStamp = () => { your code }
```